docs/generic_geometry.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>I think this cell especially would benefit from a style that seems to be fashionable now, where you start a new line after an opening <code>{[(</code> bracket and indent the contents less:</p><p> </p><pre class="ql-syntax" spellcheck="false">simple_config = {
    'pixel_size': pixel_size,
    'corner_coordinates': [
        pixel_size * np.array([5, 1, 0]),
        ...
    ],
    ...
}
</pre><p> </p><p>It would also be good to add a 'now that you've created a geometry object, here are things you can do with it' cell at the end, like the detector-specific example notebooks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/European-XFEL/EXtra-geom/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done